### DISCUSSION TITLE
Duda sobre aumento de plazo

### DISCUSSION THREAD

**Discussion by MaxAl100** [IIC2133-PUC/2024-1#161] [Tarea 2] [answered]
Tengo una duda respecto al cambio de fecha final de entrega para esta segunda tarea, ya que algunos alumnos (yo incluido) trabajaron con tiempo y se esforzaron para tener la tarea lo más finalizada posible para la fecha miércoles (este fin de semana estuve más de 18 horas trabajando en la tarea) y ahora el cambio, 2 días antes de la fecha de entrega, con otros ramos que tener en consideración, no me es posible seguir trabajando en la tarea del ramo (los días extra no me sirven personalmente).

Entiendo que no se pueden sacar estos días extra y son de mucha utilidad para la gran mayoría de los alumnos, pero quería saber si existiera la posibilidad de dar décimas extra a quienes no suban nada luego de la fecha de entrega el miércoles o finalizar el ranking en esta fecha.

**Comment by tomastrivino** [comment]
Es broma cierto?

**Comment by k-i-r-b-y** [comment]
lol

**Comment by CataAlamos** [comment, marked as answer]
Hola! Lamentablemente no vamos a dar décimas adicionales, el propósito del plazo era para ayudar a la mayoría de los alumnos.
